lib/import_js/importer.rb

```diff
@@ -130,6 +145,16 @@ def message(str)
       @editor.message("ImportJS: #{str}")
     end
 
+    # @param numbers [Set]
+    # @param range [Range]
+    # @return [Boolean]
+    def any_numbers_within_range?(numbers, range)
+      numbers.each do |number|
+        return true if range.include?(number)
+      end
+      false
+    end
```

Comment:
This could have been inlined with the other stuff, I found the indirection made it harder to follow. No big deal though.